### DISCUSSION TITLE
how to use vars_combination

### DISCUSSION THREAD

**Discussion by huang-x-h** [apache/apisix#6749] [Q&A] [answered]
When we use upstream use chash balancer, then choose hash on vars_combination, but how to config? No document find 😢

**Comment by soulbird** [comment, marked as answer]
reference： https://github.com/apache/apisix/blob/master/docs/en/latest/admin-api.md#upstream

Related Instructions：

The following should be considered when setting the `hash_on` value:

- When set to `vars`, a `key` is required. The value of the key can be any of the [Nginx variables](http://nginx.org/en/docs/varindex.html) without the `$` prefix.
- When set to `header`, a `key` is required. This is equal to "http\_`key`".
- When set to `cookie`, a `key` is required. This key is equal to "cookie\_`key`". The cookie name is case-sensitive.
- When set to `consumer`, the `key` is optional and the key is set to the `consumer_name` captured from the authentication Plugin.
- When set to `vars_combinations`, the `key` is required. The value of the key can be a combination of any of the [Nginx variables](http://nginx.org/en/docs/varindex.html) like `$request_uri$remote_addr`.
- When no value is set for either `hash_on` or `key`, the key defaults to `remote_addr`.
